SnM/SnM_Window.cpp

```diff
@@ -232,9 +232,7 @@ void GetVisibleTCPTracks(WDL_PtrList<MediaTrack>* _trList)
 	for (int i = 1; i <= GetNumTracks(); i++) // skip master
 	{
 		MediaTrack* tr = CSurf_TrackFromID(i, false);
-		TCPY = static_cast<int>(GetMediaTrackInfo_Value(tr, "I_TCPY"));
-		TCPH = static_cast<int>(GetMediaTrackInfo_Value(tr, "I_TCPH"));
-		GetTrackGap(TCPH, &topGap, &bottomGap);
+		TCPH = GetTrackHeightAndOffsetWithSpacers(tr, &TCPY, &topGap, &bottomGap);
```

Comment:
This breaks the "Toggle offscreen item selection" actions' toggle state by changing `TCPY` from relative (to the visible area) to absolute (`I_TCPY + si.nPos`). Also spacers should not be included here.

```diff
diff --git a/SnM/SnM_Window.cpp b/SnM/SnM_Window.cpp
index eae2460c..c41e5bbb 100644
--- a/SnM/SnM_Window.cpp
+++ b/SnM/SnM_Window.cpp
@@ -232,7 +232,8 @@ void GetVisibleTCPTracks(WDL_PtrList<MediaTrack>* _trList)
 	for (int i = 1; i <= GetNumTracks(); i++) // skip master
 	{
 		MediaTrack* tr = CSurf_TrackFromID(i, false);
-		TCPH = GetTrackHeightAndOffsetWithSpacers(tr, &TCPY, &topGap, &bottomGap);
+		TCPY = static_cast<int>(GetMediaTrackInfo_Value(tr, "I_TCPY"));
+		TCPH = GetTrackHeight(tr, nullptr, &topGap, &bottomGap);
 		if ((TCPY + TCPH - bottomGap > 0) && (TCPY + topGap < arrangeHeight))
 			_trList->Add(tr);
 	}
```

Reply:
thanks, fixed and squash/force-pushed to branch